Gaijinizer patch/Map468.txt

```diff
@@ -3,360 +3,431 @@
+Oh my, you've become so cute in the short moment
+I took my eyes off you... ♡
 
+Fufu... let me give you a little squeeze. ♡#Ron
 > END STRING
 
 > BEGIN STRING
 大丈夫よ、食べたりしないから♡
 
 ちょっと可愛がってあげるだけ…♡
-> CONTEXT: Map468/events/35/pages/0/26/Dialogue/highslag_fc1 < UNTRANSLATED
-> CONTEXT: Map541/events/35/pages/0/25/Dialogue/highslag_fc1 < UNTRANSLATED
+> CONTEXT: Map468/events/35/pages/0/26/Dialogue/highslag_fc1
+> CONTEXT: Map541/events/35/pages/0/25/Dialogue/highslag_fc1
+It's okay, I won't eat you or anything. ♡
 
+I just want to spoil you a little... ♡#Ron
 > END STRING
 
 > BEGIN STRING
 返り討ちにしたいところだけど
 今の私たちじゃ、あの子にも敵わないのね…
 
 逃げるわよ！！
-> CONTEXT: Map468/events/35/pages/0/30/Dialogue/enemy38 < UNTRANSLATED
-> CONTEXT: Map541/events/35/pages/0/29/Dialogue/enemy38 < UNTRANSLATED
+> CONTEXT: Map468/events/35/pages/0/30/Dialogue/enemy38
+> CONTEXT: Map541/events/35/pages/0/29/Dialogue/enemy38
+As much as I'd like to turn the tables on her, we
+aren't in a position to do that right now.
 
+Stay away from her!!#Ron
 > END STRING
 
 > BEGIN STRING
 暖炉の上にスイッチがある……
 
 押してみますか？
-> CONTEXT: Map468/events/37/pages/1/2/Dialogue < UNTRANSLATED
-> CONTEXT: Map541/events/37/pages/1/2/Dialogue < UNTRANSLATED
+> CONTEXT: Map468/events/37/pages/1/2/Dialogue
+> CONTEXT: Map541/events/37/pages/1/2/Dialogue
+There's a switch above the fireplace...
 
+Try pushing it?#Ron
 > END STRING
 
 > BEGIN STRING
 レバーがある……
 
 動かしてみますか？
-> CONTEXT: Map468/events/45/pages/0/2/Dialogue < UNTRANSLATED
-> CONTEXT: Map541/events/45/pages/0/2/Dialogue < UNTRANSLATED
+> CONTEXT: Map468/events/45/pages/0/2/Dialogue
+> CONTEXT: Map541/events/45/pages/0/2/Dialogue
+There's a lever here...
 
+Try moving it?#Ron
 > END STRING
 
 > BEGIN STRING
 
 　　　　　　　　ナード男爵の手記
-> CONTEXT: Map468/events/46/pages/1/2/Dialogue < UNTRANSLATED
-> CONTEXT: Map541/events/46/pages/1/2/Dialogue < UNTRANSLATED
+> CONTEXT: Map468/events/46/pages/1/2/Dialogue
+> CONTEXT: Map541/events/46/pages/1/2/Dialogue
 
+          Baron Nard's Notes#Ron
 > END STRING
 
 > BEGIN STRING
 我らが同胞、オタク男爵の家系が潰えてから
 
 もう随分と、月日がたったように思える……
-> CONTEXT: Map468/events/46/pages/1/6/Dialogue < UNTRANSLATED
-> CONTEXT: Map541/events/46/pages/1/6/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map468/events/46/pages/1/6/Dialogue
+> CONTEXT: Map541/events/46/pages/1/6/Dialogue
+It feels like it's been quite some time since the fall of
+our brethren, the house of Baron Otaku.#Ron
 > END STRING
 
 > BEGIN STRING
 天使の加護を授かり、先代が決死の想を受け継いで
 あの凶悪な『ヴァンパイア』を封印し続けてきたというのに…
 
 その仕打ちがこれか……
-> CONTEXT: Map468/events/46/pages/1/11/Dialogue < UNTRANSLATED
-> CONTEXT: Map541/events/46/pages/1/11/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map468/events/46/pages/1/11/Dialogue
+> CONTEXT: Map541/events/46/pages/1/11/Dialogue
+Though our previous generation, with the blessings of the
+angels, fervently carried on their duty of continuing to
+seal away the vicious "vampires"... they suffered such
+appalling mistreatment...#Ron
 > END STRING
```

Comment:
Looking at the DeepL translation and what the next box says, I think this makes more sense?
```
Our previous generations, with the blessings of the
angels, fervently carried on their duty of continuing to
seal away the vicious "vampires"... 
And *this* is what they repay us with.
```